src/config.ts

```diff
@@ -340,6 +352,12 @@ export class ConfigurationReader implements vscode.Disposable {
     get ctestDefaultArgs(): string[] {
         return this.configData.ctestDefaultArgs;
     }
+    get packEnvironment() {
```

Comment:
What is the difference between `cpackVars` and `cpackArgs`? It feels like these should be the same settings. 

Also, I don't see these defined in the package.json.

Reply:
Exactly, I removed cpackVars in my recent update.